theia-cpp-docker/Dockerfile

```diff
@@ -101,16 +101,23 @@ RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
                        clang-tools-$LLVM \
                        clangd-$LLVM \
                        clang-tidy-$LLVM \
+                       gcc-multilib \
+                       g++-multilib \
                        gdb && \
-    ln -s /usr/bin/clangd-$LLVM /usr/bin/clangd && \
+    ln -s /usr/bin/clang-$LLVM /usr/bin/clang && \
+    ln -s /usr/bin/clang++-$LLVM /usr/bin/clang++ && \
+    ln -s /usr/bin/clang-cl-$LLVM /usr/bin/clang-cl && \
+    ln -s /usr/bin/clang-cpp-$LLVM /usr/bin/clang-cpp && \
     ln -s /usr/bin/clang-tidy-$LLVM /usr/bin/clang-tidy && \
-    rm -rf /var/lib/apt/lists/*
```

Comment:
Why remove this "cleanup" line?

Reply:
I removed the line at this position because I thought it would be a good idea to do the cleanup only once after the last install with apt:

```
# Theia application
##Needed for node-gyp, nsfw build
RUN apt-get update && apt-get install -y python build-essential && \
  apt-get clean && \
  apt-get autoremove -y && \
  rm -rf /var/cache/apt/* && \
  rm -rf /var/lib/apt/lists/* && \
  rm -rf /tmp/*
```

Reply:
This way of doing it once at the end is good for the final tidiness of the image's file system, but IIUC has a drawback: unless unwanted files are removed from the image in the same docker command (i.e. same image layer) they were created, they will still be in the image but hidden. 

So ideally such cleanups would happen for every `apt-get` command. However I can't say that we've been very careful so far to do this everywhere in our images, so it's not necessarily a deal breaker for this PR.  

WDYT?

Reply:
This is a good point which have not thought about. It will have only a minimum effect on build times, so I'm going to copy the cleanup-block after each apt-get usage.

Reply:
> Changes made and pushed, the size of the final image stays the same.

That part is a bit unexpected. I wonder why? Any theories? I wonder if you were to remove all these cleanups related to installing Debian packages, including the final one, would the image size increase?